server.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const EXPRESS = require('express');
+const PORT = 3000;
+const APP = EXPRESS();
+
+APP.use(EXPRESS.static('public'));
+
+APP.listen(PORT);
```

Comment:
You should include a `console.log` statement so that you know which port the server is running on, just in case.